graph_def_editor/visualization/__init__.py

```diff
@@ -0,0 +1,2 @@
+"""Dummy __init__.py to keep pytest happy."""
```

Comment:
Can you please add a copyright header to the top of this file? You can copy it from any other file in the project.